config.ru

```diff
@@ -1,4 +1,5 @@
 # This file is used by Rack-based servers to start the application.
 
-require ::File.expand_path('../config/environment', __FILE__)
-run Alaveteli::Application
+require_relative 'config/environment'
+
+run Rails.application
```

Comment:
Oh this was an internal note to go back and check whether this change is okay without doing something inside `config/application.rb`, but since I can boot the app I guess its fine!